### ISSUE TITLE
Resume

### ISSUE THREAD

**Issue by Mynany** [Mynany/articles#1]
---
title: about
date: 2019-01-19 17:41:05
---

# <center>Yumeng Yang</center>
 <center>(312)241 0891 | yyang244@uic.edu | [LinkedIn](https://www.linkedin.com/in/yumengyang2/) | [Github](https://github.com/Mynany) </center>

##### EDUCATION
---
__University of Illinois at Chicago (UIC)__                                                 Master of Science in Computer Science Chicago, IL                                                                                                          Expected: December 2018 
__Chongqing University__                                                                    Bachelor of Engineering in Internet of Things         Chongqing, China                                                                                               June 2017  

##### TECHNICAL SKILLS 
---
__Programming language__: C++, C, Python, HTML/CSS, Java, MySQL, JavaScript
__Other__: Linux, sklearn, VueJS, NodeJS, React Native, Git, ES6, React, Redux, Spring boot, Ajax, JSON

##### PROJECTS
---
__Recruitment Web-App__
•    Implemented an online recruitment web-app with React, Redux, MongoDB and Node.js.
•    Used Redux to manage data and state, used React Router to build the router system. Applied axios to achieve asynchronous message transfer mechanism and interceptor.
•    Used connect and Provider to connect react components with redux store, used Redux middlewares Thunk for asynchronous dispatch of actions.
•    Implemented backend with Express and MongoDB. Implemented bi-directional communication between employers and applicants with Socket.io.

__Events recommendation__
•    Designed FunC web app where users can browse events and join groups under each event.
•    Built a crawler which scrapes dedupe latest events in Chicago periodically.
•    Designed an offline training tool for event key words extraction with Python.
•    Implemented recommendation function by matching users’ query and likes with events’ keywords
•    Implemented front end with Vue.JS

__Search Engine__
•    Developed a Crawler with Scrapy-Redis. 
•    Used selenium and phantomjs to implement browser automation and scrape dynamic websites
•    Loaded data into elasticsearch, achieved searching, search recommendation, storing searching history and popular searching history by elasticsearch.
•    Implemented pause and restart of scrapy, Deployed scrapy by scrapyd
•    Built a search engine by Django.

__Sentiment analysis__
•    Implemented sentiment analysis system with Python to generate the overall users’ sentiment toward a product based on comments they posted online.
•    Preprocessed data by stemming, marking negation and removing stopwords with pandas and numpy.
•    Trained word embeddings on training set by doc2vec.
•    Applied classification model Naïve Bayes, GBDT, SVM, RandomForest and NeutrulNetwork. Polished SVM model performance, improved overall accuracy of the system from 55.9% to 74.1%

##### INDUSTRIES
---
__Next generation technology Inc__                                                                                               Apr 2019 – now
•	Full stack development in agile development environment
•	Refactored website with React, NodeJS, implemented server side rendering, optimized SEO
•	Developed time portal system with java, tomcat, MySQL. 

__Chinasoft International __                                                                                                           Jun 2017 - Jul 2017
•    Developed Android Game “Plane Wars” with C# and Unity3D including game scene design, object collision design and object action design.

##### others
---
I love music!!!!!
